### PR TITLE
feat(user-notification): Firebase Credentials Setup for Local Development

### PR DESCRIPTION
**Pull request by rafnarnason**
Fixes these types of strings for local development env.secret file so the app runs locally without manual edits to the env value

## Before: 
export FIREBASE_CREDENTIALS='{\n  \"type\": \"service_account\",\n  \"project_id\":
## After: 
export FIREBASE_CREDENTIALS='{
  ...
  "type": "service_account",
  "project_id": "mobile-app",
  ...
 }'

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

## Summary by CodeRabbit

- **Bug Fixes**
	- Improved the escaping of secret values in console logs for better display of environment variables.

### CODE REVIEW

**Review by kksteini — APPROVED** [review]

File: apps/services/user-notification/README.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ Get secrets:
 yarn get-secrets user-notification
 ```
 
+> **Important**: For local development, the `FIREBASE_CREDENTIALS` environment variable needs to be cleaned of newlines. After running `get-secrets`, make sure to remove any newline characters from the `FIREBASE_CREDENTIALS` value in your environment.
```
Comment (baering) [inline apps/services/user-notification/README.md:43]: Thinking "out loud": What about keeping the dev-init as is and doing this inside [UserNotificationsConfig](https://github.com/island-is/island.is/blob/main/apps/services/user-notification/src/config.ts#L27) for local development?

Something similar to:

```typescript
let firebaseCredentials = env.required('FIREBASE_CREDENTIALS', '')

if (isRunningOnEnvironment('local')) {
  firebaseCredentials = firebaseCredentials.replace(/\n/g, '')
}

return {
  // ...
  firebaseCredentials,
  // ...
}
```

Comment (baering) [reply apps/services/user-notification/README.md:43]: Yes, it is something that was implemented 4 years ago when there was no standard way to check what environment you were running on. This was before we had a feature flag service. After ConfigCat this was phased out as it was not meant to be used as a way to "feature flag" code paths by environments.

For this use case _I think it should be fine_. But if you have a better way feel free, it was the first thing that came to my mind to showcase the idea.

Comment (eirikurn) [reply apps/services/user-notification/README.md:43]: How come it works in prod and not locally? Isn’t this just a formatting bug in our get-secrets script which we can fix?

Comment (rafnarnason) [reply apps/services/user-notification/README.md]: yes - that would sound like the right approach - developers should never have to touch these values - I have updated get-secrets and it works for this case ... just wonder if there could be any other unintended consequences

**Review by robertaandersen — APPROVED** [review]
LGTM

**Review by kksteini — APPROVED** [review]

**Review by eirikurn — APPROVED** [review]

**Pull request merged**